package.json

```diff
@@ -90,6 +90,7 @@
     "marked": "^2.0.1",
     "ng2-nouislider": "^1.8.3",
     "ngx-color": "^6.1.0",
+    "ngx-extended-pdf-viewer": "17.0.0-alpha.4",
```

Comment:
nit(non-blocking)
Any particular reason to use v17 which is in alpha as opposed to v16 stable?